index.js

```diff
@@ -25,6 +25,7 @@ function packSmartCardData(camFlags, emvTags) {
 }
 
 function NDC(config, validator, logger) {
+    this.errors = require('./errors')(config.defineError);

```

Comment:
Good catch! I forgot that I made this available to the codec through config param.